composer.json

```diff
@@ -22,5 +27,12 @@
 		"branch-alias": {
 			"dev-master": "0.7.x-dev"
 		}
+	},
+	"scripts": {
+		"csfix": "phpcbf --standard=phpcs.xml",
```

Comment:
Please use the standard names (https://github.com/wikimedia/mediawiki-tools-codesniffer#how-to-install). Notably, --standard= should not be necessary.